src/test/java/subway/line/LineAcceptanceTest.java

```diff
@@ -30,18 +41,12 @@ public class LineAcceptanceTest {
     @Test
     void createLine() {
         // when
-        Long 신사역 = 지하철역_생성("신사역");
-        Long 논현역 = 지하철역_생성("논현역");
-
-        ExtractableResponse<Response> response = 지하철노선_생성(
-            createParams("신분당선", "bg-red-600", 신사역, 논현역, 10L));
+        ExtractableResponse<Response> 신분당선_생성_응답 = 지하철노선_생성("신분당선", "bg-red-600", 지하철역_id("신사역"), 지하철역_id("논현역"), 10L);
+        assertThat(신분당선_생성_응답.statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

Comment:
아예 이만큼을 묶어도 될것이라고 생각됩니다 
```suggestion
        ExtractableResponse<Response> 신분당선_생성_응답 = 지하철노선_생성("신분당선", "bg-red-600", 지하철역_id("신사역"), 지하철역_id("논현역"), 10L);
```

안에서 http status 검증정도는 끝내는 걸로 생성을 보장해도 좋을 것 같아요 😄